src/Document.php

```diff
@@ -231,7 +231,6 @@ protected function setErrors($errors)
     protected function getDomDocumentFromString($stringDocument)
     {
         libxml_use_internal_errors(true);
-        libxml_disable_entity_loader(true);
```

Comment:
https://github.com/php/php-src/blob/5998b2a3a65042dc7fc4f70945dd72e4e258500f/UPGRADING#L872-L875

Reply:
Please have a look at the solution in laminas-feed: https://github.com/laminas/laminas-feed/pull/29

Reply:
Sorry, of course it must be kept on PHP < 8.0, fixed

Reply:
I'm sorry, my comment was not clear. The check should be moved to a separate method to avoid duplication of logic. Like in laminas-feed:

https://github.com/laminas/laminas-feed/blob/221328023f3eb1ffd2e57f4db18abd4823b87440/src/Reader/Reader.php#L102-L108

Reply:
Sorry, done